test/components/b-u-v.hide-footer_test.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Blockcerts Universal Verifier - API certificate test</title>
+  <script src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script src="../../node_modules/web-component-tester/browser.js"></script>
+
+  <script type="module" src="../../src/blockcerts-verifier/index.js"></script>
+</head>
+
+<body>
+  <script src="./helpers/redux-hack.js"></script>
+  <script src="./helpers/wait.js"></script>
+  <test-fixture id="non-actionable">
+    <template>
+      <blockcerts-verifier src="../fixtures/valid-certificate-example.json" hide-footer></blockcerts-verifier>
+    </template>
+  </test-fixture>
+
+  <script type="module">
+    describe('Blockcerts Universal Verifier test suite', function () {
+      describe('hiding the footer of a certificate', function () {
+        describe('given the src is of a valid blockcerts definition', function () {
+          describe('and the allow-download flag is off', function () {
+            it('the download link should not be available', async function () {
+              const el = fixture('non-actionable').shadowRoot.querySelectorAll('buv-raw')[0];
+              const menu = el.shadowRoot.querySelectorAll('buv-action-menu')[0];
+              await wait(200);
+              const downloadLinkComponent = menu.shadowRoot.querySelectorAll('buv-download-link');
+              expect(downloadLinkComponent.length).to.equal(0);
```

Comment:
same as above, these tests do not verify https://github.com/blockchain-certificates/blockcerts-verifier/pull/105/files#diff-5a725866e669b3dd7372d9abe2392180R24

We do not have a test for the footer yet, so I can't point you to a correct example, ubt it will be a good opportunity to add one. I think it should be very similar to this test, except you'd looking to find this element `buv-verify-other-certificate` here: https://github.com/blockchain-certificates/blockcerts-verifier/pull/105/files#diff-5a725866e669b3dd7372d9abe2392180R31.

Note that the path will be different that the one above.

Please consider adding a test to verify that the element is here in case the flag is off.